src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -733,10 +733,10 @@ public new CSharpCompilation RemoveSyntaxTrees(IEnumerable<SyntaxTree> trees)
                     var loadedSyntaxTreeMap = syntaxAndDeclarations.GetLazyState().LoadedSyntaxTreeMap;
                     if (SyntaxAndDeclarationManager.IsLoadedSyntaxTree(tree, loadedSyntaxTreeMap))
                     {
-                        throw new ArgumentException(string.Format(CSharpResources.SyntaxTreeFromLoadNoRemoveReplace, tree), $"{nameof(trees)}[{i}]");
+                        throw new ArgumentException(string.Format(CSharpResources.SyntaxTreeFromLoadNoRemoveReplace), $"{nameof(trees)}[{i}]");
```

Comment:
>$"{nameof(trees)}[{i}]" [](start = 118, length = 23)

The extra arguments can be removed, for string where `{0}` was removed.

Reply:
The remaining arguments are for the `ArgumentException` constructor, not for `string.Format`. Though the call to `string.Format` can be removed, so I just did that.